### ISSUE TITLE
GoPro app replacement

### ISSUE THREAD

**Issue by garydelarosausername** [KonradIT/GoProStream#12]
I don't know if this is the place to suggest ideas, but a GoPro app replacement would be nice, very nice! I'm up to help. How do I start to be able to help make this a reality? Haha! It's funny! But really, I think this is neat! And if it helps that there be another person to work on this, I have quite some time on my hands. I'm down to help!

I have the GoPro 5 Black. And so I'm looking forward that the replacement will work with my camera.

**Comment by KonradIT** [comment]
http://github.com/konradit/camcontrol

Don't have time to maintain it.

**Comment by garydelarosausername** [comment]
Oh, I forgot to state the reason of why even this post. The GoPro app replacement that I want is the one that can be used from a laptop/desktop/PC. What I want is a GoPro app desktop version basically. And then, on top of that, being able to project it to a bigger screen say a TV via HDMI. I think that's really cool!

**Comment by KonradIT** [comment]
http://github.com/konradit/gopro-control

Preview sizes are not HD so casting to a TV is a bad idea.

**Comment by garydelarosausername** [comment]
@KonradIT Point there. Somehow I'm thinking casting/outputting preview to a large screen like TV might somehow be usable, like brainstorming with a couple of people on video angles and whatnot.

**Comment by KonradIT** [comment]
Yup, you can already do that with GoProStream anyway, just connect your computer to a TV and move the window to the TV

**Comment by garydelarosausername** [comment]
Cool! Thanks a lot!
